packaging/linux/README.mount

```diff
@@ -0,0 +1,8 @@
+This is where you can access the Keybase File System (KBFS)!  See
```

Comment:
Very open to suggestions about the wording here, and including other details.

Reply:
@malgorithms specifically you might care about the contents of this file, which will appear to the user under `/keybase` before they run `run_keybase` for the first time.

Reply:
for now, this wording is fine with me